fastly/block_fastly_service_v1_package.go

```diff
@@ -0,0 +1,108 @@
+package fastly
+
+import (
+	"fmt"
+	"log"
+
+	gofastly "github.com/fastly/go-fastly/fastly"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+type PackageServiceAttributeHandler struct {
+	*DefaultServiceAttributeHandler
+}
+
+func NewServicePackage() ServiceAttributeDefinition {
+	return &PackageServiceAttributeHandler{
+		&DefaultServiceAttributeHandler{
+			key: "package",
+		},
+	}
+}
+
+func (h *PackageServiceAttributeHandler) Register(s *schema.Resource) error {
+	s.Schema[h.GetKey()] = &schema.Schema{
+		Type:     schema.TypeList,
```

Comment:
Whats the reasoning for this being a List which then limits its size to 1? Is it too support a future in which there may multiple packages? Something else? Lastly, whats the benefit/reasoning of a list over a set?

Reply:
Makes sense, thank you 👍